config/etc/system-image/writable-paths

```diff
@@ -13,6 +13,7 @@
 /tmp                                    none                    temporary   none        defaults
 /mnt                                    none                    temporary   none        defaults
 /media                                  none                    temporary   none        defaults
+/boot/androidboot                       auto                    persistent  none        none
```

Comment:
why dont we have it ? not having it will mean extra changes to ubuntu-image, the gadget spec, snapd and other parts of the system. i'd just say we require one like in all other cases, carrying the above mentioned config and the boot.img copied from the kernel snap. that way we avoid changing to much existing code. after all we define our partitioning after the bootloader setup currently. if we want to start supporting non-standard (a standard we define ourselves atm) setups later this is fine, but for the moment i'd go with the least changes we can do to get a basic setup going. 

also, regarding the snap_try var and friends, if you write to a filesystem instead of using abootimg to directly modify the boot partition cmdline this means you have caches involved while direct writing to the boot partition will be atomic and less prone to something like a power outage during upgrade.